BaseFilter/sections/antiadblock.txt

```diff
@@ -273,7 +273,7 @@ macbed.com#%#//scriptlet('prevent-setInterval', 'clientHeight')

```

Comment:
Add `itpxhub.com` at the beginning of the rule.
Also add `@@||itpxhub.com^$generichide` to the content_blockers.txt - antiblock.org g207 near this rule `@@||hyogo.ie-t.net^$generichide`.